xrmocap/human_perception/bbox_detection/__init__.py

```diff
@@ -0,0 +1,7 @@
+from .mmdet_detector import MMdetDetector, process_mmdet_results
+from .mmtrack_detector import MMtrackDetector, process_mmtrack_results
+
+__all__ = [
+    'MMdetDetector', 'MMtrackDetector', 'process_mmdet_results',
```

Comment:
`process_mm*_results` is not so good for users to read. Advanced developers can read them in code.

Reply:
All right. Docstring of the primitive code does look good.